tests/basic/pure_test.go

```diff
@@ -6173,3 +6173,129 @@ var _ = Describe("{VerifyPoolCreateInProperZones}", func() {
 		AfterEachTest(contexts)
 	})
 })
+var _ = Describe("{RebootAllWorkerNodesandCheckPX}", func() {
+	/*
+	   https://purestorage.atlassian.net/browse/PTX-24005
+	   1.Deploy Applications
+	   2.Validate Applications are Deployed
+	   3.Collect the existing Volume status (state,status) before stopping the PX
+	   4.Stop portworx on all nodes
+	   5.Restart all the worker nodes
+	   6.Make sure Both portworx and node are up.
+	   7.Collect the  Volume status (state,status) after restarting the PX and compare with the status before stopping the PX
+	   8.Validate the Applications are running
+	*/
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootAllWorkerNodesandCheckPX",
```

Comment:
seems test is doing px restart and not node reboots. please correct the test name accordingly